text/0000-north-star.md

```diff
@@ -0,0 +1,466 @@
+stable will be estimated as well. Along the way, teams will be expected to
+maintain _tracking issues_ that communicate progress toward the project's goals.
+
+At the end of the year we will deliver a public facing retrospective, which is
+intended as a 'rallying point'. Its primary purposes are to create anticipation
+of a major event in the Rust world, to motivate (rally) contributors behind the
+goals we've established to get there, and generate a big PR-bomb where we can
+brag to the world about what we've done. It can be thought of as a 'state of the
+union'. This is where we tell Rust's story, describe the new best practices
+enabled by the new features we've delivered, celebrate those contributors who
+helped achieve our goals, honestly evaluate our performance, and look forward to
+the year to come.
+
+## Summary of terminology
+
+Key terminology used in this RFC:
+
+- _problem statement_ - A description of a major issue facing Rust, possibly
+  spanning multiple teams and disciplines. We decide these together, every year,
+  so that everybody understands the direction the project is taking. These are
+  used as the broad basis for decision making throughout the year, and are
+  captured in the yearly "north star RFC", and tagged `R-problem-statement`
+  on the issue tracker.
+
+- _goal_ - These are set by individual teams quarterly, in service of solving
+  the problems identified by the project. They have estimated deadlines, and
+  those that result in stable features have estimated release numbers. Goals may
+  be subdivided into further discrete tasks on the issue tracker. They are
+  tagged `R-goal`.
+
+- _retrospective_ - At the end of the year we deliver a retrospective report. It
+  presents the result of work toward each of our goals in a way that serves to
+  reinforce the year's narrative. These are written for public consumption,
+  showing off new features, surfacing interesting technical details, and
+  celebrating those who contribute to achieving the project's goals and
+  resolving it's problems.
+
+- _quarterly milestone_ - All goals have estimates for completion, placed on
+  quarterly milestones. Each quarter that a goal remains incomplete it must be
+  re-triaged and re-estimated by the responsible team.
+
+## The big planning cycle (problem statements and the north star RFC)
+
+The big cycle spans one year. At the beginning of the cycle we identify areas of
+Rust that need the most improvement, and at the end of the cycle is a 'rallying
+point' where we deliver to the world the results of our efforts. We choose
+year-long cycles because a year is enough time to accomplish relatively large
+goals; and because having the rallying point occur at the same time every year
+makes it easy to know when to anticipate big news from the project. Being
+calendar-based avoids the temptation to slip or produce feature-based releases,
+instead providing a fixed point of accountability for shipping.
```

Comment:
@nrc No special releases besides the retrospective. If we ever get to point where we are ready to do LTS or larger-scope "platform" releases that could fill a similar role.